File: src/main/java/com/czh/jiaowuchu/CookieUtil.java
package com.czh.jiaowuchu;

import okhttp3.Cookie;

import java.util.HashMap;

public class CookieUtil {

    public static String id = "";
    public static String cookie = "";
    public static String jwchLoginCookie = "";
    public static HashMap newJwchCookies = new HashMap<String, Cookie>();
    public static Long newJwchLastUpdateTime = 0L;
    public static String authorizationCookie = "";
    public static HashMap graduateCookies = new HashMap<String, Cookie>();
    public static Long lastUpdateTime = 0L;
    public static Long jwtLastUpdateTime = 0L;
    public static Long graduateLastUpdateTime = 0L;
    public static String emptyRoomViewState = "";
    public static String emptyRoomEventValidation = "";
    public static String yiBanCookie = "";
    public static String libCookie = "";
    public static String experimentCookie = "";
    public static HashMap eCardCookies = new HashMap<String, Cookie>();
}

File: src/main/java/com/czh/jwc/CookieUtil.kt
package jiaowuchu

import okhttp3.Cookie

/**
 *单例临时保存Cookie相关
 */
object CookieUtil {
    var id: String = ""
    var cookie: String = ""
    var jwchLoginCookie: String = ""
    var newJwchCookies = mutableMapOf<String, Cookie>()
    var newJwchLastUpdateTime: Long = 0
    var authorizationCookie: String = ""
    var graduateCookies = mutableMapOf<String, Cookie>()
    var lastUpdateTime: Long = 0
    var jwtLastUpdateTime: Long = 0
    var graduateLastUpdateTime: Long = 0
    var emptyRoomViewState: String = ""
    var emptyRoomEventValidation: String = ""
    var yiBanCookie: String = ""
    var libCookie: String = ""
    var experimentCookie: String = ""
    var eCardCookies = mutableMapOf<String, Cookie>()

    fun clear() {
        id = ""
        cookie = ""
        jwchLoginCookie = ""
        newJwchCookies.clear()
        newJwchLastUpdateTime = 0
        authorizationCookie = ""
        graduateCookies.clear()
        jwtLastUpdateTime = 0
        graduateLastUpdateTime = 0
        lastUpdateTime = 0
        yiBanCookie = ""
        libCookie = ""
        experimentCookie = ""
        eCardCookies.clear()
    }
}